modules/develop/partials/run-transforms.adoc

```diff
@@ -245,9 +246,122 @@ func filterValidJson(e transform.WriteEvent, w transform.RecordWriter) error {
 	return nil
 }
 ```
+--
+
 
+Rust::
++
 --
+
+[tabs]
+====
+Identity transform::
++
+```rust
+use anyhow::Result;
+use redpanda_transform_sdk::*;
+
+// This example shows the basic usage of the crate:
+// This transform does nothing but copy the same data from an
+// input topic to an output topic.
+fn main() {
+    // Make sure to register your callback and perform other setup in main
+    on_record_written(my_transform);
+}
+
+// This will be called for each record in the source topic.
+//
+// The output records returned will be written to the destination topic.
+fn my_transform(event: WriteEvent, writer: &mut RecordWriter) -> Result<()> {
+    writer.write(event.record)?;
+    Ok(())
+```
+
+
+Transcoder transform::
++
+```rust
+// Copyright 2024 Redpanda Data, Inc.
```

Comment:
Ok to remove copyright block of comments?

Reply:
Yes